### DISCUSSION TITLE
如何在 Form.List 校验成功的时候, 返回一个成功消息值, 展示在 help 中?

### DISCUSSION THREAD

**Discussion by xuoutput** [ant-design/ant-design#39206] [Q&A]
用 Form.List ,的 rule, 就不太想用[自定义的校验](https://4x.ant.design/components/form-cn/#components-form-demo-validate-static)
有什么方便的方式来实现么

## Link to minimal reproduction
https://codesandbox.io/s/dong-tai-zeng-jian-biao-dan-xiang-antd-4-24-4-shu-ru-zheng-que-de-shi-hou-zhan-shi-zheng-que-de-message-dnmwo3?file=/demo.tsx

## What is expected?
在输入的不是 hello 的时候, 展示报错信息
**在输入框中输入 hello 的时候, 底部help 值展示 world**
<img width="516" alt="image" src="https://user-images.githubusercontent.com/16385443/205317370-334a7149-afa4-4a95-8335-844828ab04d7.png">

**Comment by zombieJ** [comment]
一般来说都是：
<img width="99" alt="截屏2022-12-02 22 46 34" src="https://user-images.githubusercontent.com/5378891/205319072-2a331cc1-c177-4fb3-b8b9-0c643a7fccbe.png">


特意搞个文字还挺奇怪的，那就是你说的 help 了

**Reply by xuoutput** [reply]
> 一般来说都是： <img alt="截屏2022-12-02 22 46 34" width="99" src="https://user-images.githubusercontent.com/5378891/205319072-2a331cc1-c177-4fb3-b8b9-0c643a7fccbe.png">
> 
> 特意搞个文字还挺奇怪的，那就是你说的 help 了

是的, 有办法在 rule validator 这里返回 resole 信息放在 help 么, 类似错误信息的 new Error 这种功能
如果要自己自定义校验成功时的提示文案, 可能就不能用 rules 功能了

**Comment by xuoutput** [comment]
而且如果用自定义的校验 validateStatus + help, 有办法将触发条件改成和 Form 的 form.validateFields() 一样么
有什么变量可以获取当前是否触发了 form.validateFields() 么

**Comment by xuoutput** [comment]
发现自定义 如果用自定义的校验 validateStatus + help  还需要在提交的地方再写一遍 校验, 更难受了

**Comment by xuoutput** [comment]
@zombieJ 😂 大佬这块有什么好的建议么, 确实需要调整一点点自定义校验消息
